src/odemis/util/spot.py

```diff
@@ -201,9 +201,10 @@ def FindCenterCoordinates(image, smoothing=True):
     w = numpy.sqrt(dI2 / numpy.hypot(ik - i0, jk - j0))
 
     # Solve the linear set of equations in a least-squares sense.
-    # Note: rcond set to -1 to suppress warnings on numpy 1.11 -> 1.14. It maybe
-    # could be set to None with newer numpy (or entirely removed).
-    ic, jc = numpy.linalg.lstsq(numpy.vstack((w * a, w * b)).T, w * c, rcond=-1)[0]
+    # Note: rcond is set explicitly to have correct and consistent behavior,
+    #       independent on numpy version.
+    rcond = numpy.finfo(numpy.float64).eps * max(m, n)
+    ic, jc = numpy.linalg.lstsq(numpy.vstack((w * a, w * b)).T, w * c, rcond)[0]
```

Comment:
The changes related to this file don't seem to have anything to do with anamorphosis. Please drop them (and submit them via another PR).